tests/integration/test_tf_symbolic.py

```diff
@@ -81,6 +82,7 @@ def test_eng_run_eval_false_measurements_are_tensors(self, setup_eng):
         val = q[0].val
         assert isinstance(val, tf.Tensor)
 
+    @pytest.mark.broken('FIXME')
```

Comment:
Why is this test broken? Might be better to add something to the mark string

Reply:
It's fixed now.